### PR TITLE
[BetterStandardPrinter] Fixes #4274 #4573 Annotation callback and Route values should not be changed

### LINKED ISSUE #4274: Annotation callback changed to const
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.8.7@869441e |
| Installed as   | composer dependency                      |

## Minimal PHP Code Causing Issue

https://getrector.org/demo/96aa33a1-1472-40fa-b3f6-82914632e817#result

The callbacks in annotation are changed to const.
Example

```php
    /**
     * @Assert\Choice(callback={Invoice::class, "typeChoices"})
     */
    private ?string $type;
```

will be changed to
```php
    /**
     * @Assert\Choice(callback={Invoice::class, typeChoices})
     */
    private ?string $type;
```

## Expected Behaviour

Nothing has to be changed

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: phpstan.neon [hunk 1]
```diff
@@ -691,7 +691,7 @@ parameters:
 
         - '#Class with base "Application" name is already used in "(.*?)\\Symfony\\Component\\Console\\Application", "Symfony\\Component\\Console\\Application", "Rector\\Core\\Console\\Application"\. Use unique name to make classes easy to recognize#'
         - '#Class with base "FileNode" name is already used in "PHPStan\\Node\\FileNode", "Rector\\Core\\PhpParser\\Node\\CustomNode\\FileNode"\. Use unique name to make classes easy to recognize#'
-        - '#Class cognitive complexity for "BetterStandardPrinter" is 52, keep it under 50#'
+        - '#Class cognitive complexity for "BetterStandardPrinter" is 55, keep it under 50#'
```
Comment (TomasVotruba) [inline phpstan.neon:694]:
```suggestion
        - '#Class cognitive complexity for "BetterStandardPrinter" is \d+, keep it under 50#'
```

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Review by TomasVotruba — APPROVED** [review]

File: phpstan.neon (hunk 1 above)
Comment (samsonasik) [reply phpstan.neon]: I've moved content patch functionalities to new service : `ContentPatcher`

File: src/PhpParser/Printer/BetterStandardPrinter.php [hunk 2]
```diff
@@ -110,10 +87,15 @@ final class BetterStandardPrinter extends Standard
...
+    /**
+     * @var ContentPatcher
+     */
+    private $contentPatcher;
+
     /**
      * @param mixed[] $options
      */
-    public function __construct(CommentRemover $commentRemover, array $options = [])
+    public function __construct(CommentRemover $commentRemover, array $options = [], ContentPatcher $contentPatcher)
```
Comment (TomasVotruba) [inline src/PhpParser/Printer/BetterStandardPrinter.php:98]: Supreme :clap:

**Pull request merged**